paddlenlp/trainer/trainer.py

```diff
@@ -1752,7 +1777,12 @@ def compute_loss(self, model, inputs, return_outputs=False):
             self._past = outputs[self.args.past_index]
 
         # We don't use .loss here since the model may return tuples instead of ModelOutput.
-        loss = outputs["loss"] if isinstance(outputs, dict) else outputs[0]
+        if isinstance(outputs, dict):
+            loss = outputs["loss"]
+        elif len(outputs) == 2:
+            loss, outputs = outputs
```

Comment:
```suggestion
        elif isinstance(outputs, tuple):
            loss = outputs[0]
```

https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/trainer/trainer.py#L1714-L1719